docs/source/notebook.rst

```diff
@@ -314,8 +320,8 @@ shortcut.
 
 A notebook may be downloaded in either a ``.ipynb`` or ``.py`` file from the
```

Comment:
Suggested:

A notebook may also be downloaded into a number of different formats:
* Notebook files ``.ipynb``
* Python ``.py``
* HTML ``.html``
* Markdown ``.md``
* reStructuredText ``.rst``
* PDF via LaTex ``.pdf``